content/en/docs/instrumentation/js/benchmarks.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Benchmarks
+weight: 99
```

Comment:
Can you do a weight above 100? (see https://github.com/open-telemetry/opentelemetry.io/issues/2829), serverless has 100 right now, so either swap it or pick 101 or 110